develop/develop/settings.py

```diff
@@ -148,7 +148,7 @@
 # STYLIST_DEFAULT_CSS = STATIC_URL + "css/default.css"
 STYLIST_DEFAULT_CSS = os.path.join(MEDIA_ROOT, "css/default.css")
 STYLIST_SCSS_TEMPLATE = os.path.join(BASE_DIR, "scss/base_template.scss")
-STYLIST_IGNORE_SASS = False
+STYLIST_IGNORE_SASS = True
```

Comment:
The better approach would be to name this something like "STYLIST_USE_SASS" with the right default.  "IGNORE" is not as intuitive as "USE".